mrmap/quality/tables.py

```diff
@@ -21,5 +29,50 @@ class Meta:
         fields = ('metadata', 'config', 'passed', 'report', 'created_at', 'created_by_user', 'owned_by_org')
         prefix = 'conformity_check_run-table'
 
-    def render_report(self, record):
+    @staticmethod
+    def render_report(record):
         return Link(url=record.get_report_url(), content='Validation report').render(safe=True)
```

Comment:
Dont use `Link` from django_bootstrap_swt package. I developed this package for a while. But in tables it turned out, that the rendering process slows down, cause the rendering procedure is called for every single column which uses any `Component` from this package. 

Best practice to avoid performance issues:

```python
    from main.tables.template_code import RECORD_ABSOLUTE_LINK_VALUE_CONTENT
    validation_report = tables.TemplateColumn(template_code=RECORD_ABSOLUTE_LINK_VALUE_CONTENT, empty_values=[_("validation report")]
)

```

**Important note:**
Always use translation functions to support multi languages: https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#standard-translation

```python
from django.utils.translation import gettext as _
_("Validation report")
```